### ISSUE TITLE
Crash report: NPE on new world creation

### ISSUE THREAD

**Issue by andrewminer** [WildBamaBoy/minecraft-comes-alive#853]
- [X] I am running the latest mod versions of MCA and RadixCore.
- [] I can reproduce this issue with just MCA and RadixCore installed.
- [X] I can provide my Minecraft version and MCA version.
- [X] I have searched for the issue previously and it was either not previously reported, or previously fixed and I'm having the same problem.
- [X] I am crashing and can provide my crash report.

## Versions
Minecraft 1.10.2
MCA 5.2.3
RadixCore 2.1.3

## Issue Description
I started a new world, and before it had fully loaded, it crashed.

## Reproduce Steps
Start a new world.

## Additional Information
```
java.lang.NullPointerException: Unexpected error
	at mca.core.MCA.getPlayerData(MCA.java:700)
	at mca.packets.PacketPlayerDataC.processOnGameThread(PacketPlayerDataC.java:55)
	at radixcore.network.AbstractPacketHandler.processPackets(AbstractPacketHandler.java:161)
	at mca.core.forge.EventHooksFML.clientTickEventHandler(EventHooksFML.java:135)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_239_EventHooksFML_clientTickEventHandler_ClientTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185)
	at net.minecraftforge.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:344)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1851)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1055)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by andrewminer** [comment]
Here's the full log: 
[latest.log.zip](https://github.com/WildBamaBoy/minecraft-comes-alive/files/1126700/latest.log.zip)

**Comment by WildBamaBoy** [comment]
Player data is now handled differently in 1.12. Try 1.12 and see if the same issue occurs

**Comment by andrewminer** [comment]
I'm afraid that won't help me as I'm playing with a rather substantial modpack for 1.10.2.  Many of the mods in my modpack aren't available for 1.12, yet.  Any hope of a backport for 1.10.2?
